src/components/plots/styling/volcano/DisplayLabels.jsx

```diff
@@ -0,0 +1,48 @@
+import _ from 'lodash';
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Slider, Form, Space,
+} from 'antd';
+
+const VolcanoDimensionsRangeEditor = (props) => {
```

Comment:
this doesn't sounds right -- needs to be checked

Reply:
renamed to `DisplayLabels`